src/pybind/mgr/dashboard/module.py

```diff
@@ -421,18 +421,24 @@ def set_rgw_credentials(self):
         return 0, 'RGW credentials configured', ''
 
     @CLIWriteCommand("dashboard set-login-banner")
-    def set_login_banner(self, mgr_id: Optional[str] = None, inbuf: Optional[str] = None):
+    def set_login_banner(self, inbuf: str):
+        '''
+        Set the custom login banner read from -i <file>
+        '''
         item_label = 'login banner file'
         if inbuf is None:
             return HandleCommandResult(
                 -errno.EINVAL,
                 stderr=f'Please specify the {item_label} with "-i" option'
             )
-        set_login_banner_mgr(inbuf, mgr_id)
+        set_login_banner_mgr(inbuf)
```

Comment:
@Sarthak0702 Changes LGTM, But can add the CLI output of your local test changes?

Reply:
```
[root@ceph ceph]# ceph dashboard set-login
2022-06-21T18:18:02.888+0000 7f38caf66700 -1 WARNING: all dangerous and experimental features are enabled.
2022-06-21T18:18:02.893+0000 7f38caf66700 -1 WARNING: all dangerous and experimental features are enabled.
no valid command found; 2 closest matches:
dashboard set-login-credentials <username>
dashboard set-login-banner
Error EINVAL: invalid command
```

Reply:
and for `ceph dashboard --help` :
```
dashboard get-login-banner                                                     Get the custom login banner text
dashboard set-login-banner                                                     Set the custom login banner read from -i <file>
dashboard unset-login-banner                                                   Unset the custom login banner
```

Reply:
Thanks @Sarthak0702 LGTM.